### DISCUSSION TITLE
Related to Pekko Persistence

### DISCUSSION THREAD

**Discussion by BUNNY-1269** [apache/pekko#1401] [Q&A]
Pekko offers two types of persistence: Durable State and Event Sourcing. In case of a failure or restart, will the actor’s state be restored in both cases or only when using Event Sourcing?

In my application, I used Durable State and PostgreSQL to store the actor’s state, but when I restart my pod, the actor’s state is not being restored. Can anyone help me with this issue?

**Comment by Roiocam** [comment]
> In case of a failure or restart, will the actor’s state be restored in both cases

Both

> but when I restart my pod, the actor’s state is not being restored

can you post some code that how to use the Persitence?
